run_test.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+./rebar3 ct --sname=ct1
```

Comment:
There is no `rebar3` in this repo. I would just do:

```suggestion
rebar3 ct --sname=ct1
```